### DISCUSSION TITLE
Second page not working :/

### DISCUSSION THREAD

**Discussion by ElMameri7** [community/community#22654] [Other Feature Feedback, Questions, & Ideas]
<p>Hi!<br>
i’m new to github. I uploaded 2 html pages + some css , some imgs and scripts , the first page works just fine , but when i click on the second page it doesn’t work, it just give me the 404 error.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/7midaLkesal/7midaLkesal.github.io" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/70650915/181072488-cdb488d0-9075-49ec-848d-a0ca887d5c8f.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/7midaLkesal/7midaLkesal.github.io" target="_blank" rel="nofollow noopener">7midaLkesal/7midaLkesal.github.io</a></h3>

<p>Contribute to 7midaLkesal/7midaLkesal.github.io development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>help please .</p>

**Comment by thinkverse** [comment, marked as answer]
<p>Hi there. 👋</p>
<p>I found three issues, one which can be solved two ways, first your submissions page, GitHub pages resolved the routes in a case-sensitive manner, meaning <code>submission.html</code> isn’t the same as <code>submission.HTML</code>, so one way to fix that is to either change the <code>href</code> in your html files to point to <code>submission.HTML</code> or simply change the file extension to the lowercase <code>html</code>.</p>
<p>And the same will go for all your files, for e.g. your <code>About.html</code> will match that file name, not the fully lowercase version, speaking of your about page, the anchor tags reference is wrong there on <strong>Line 17</strong> in your <em>index.html</em> <code>&lt;a href="BassSlayer About"&gt;</code> that should point to your filename <code>href="filename.html</code>.</p>
<p>Thirdly and last one, <code>BassSlayer Submission.HTML</code>, uppercase extension and space in the filename, spaces in the URL bar will translate into <code>%20</code>, which is the encoding for a single space, so no spaces in the filenames, if you need to simulate a space between words or characters, you can use a dash <code>-</code>, that’s a pretty normal standard. :+1:</p>
<p>Hope that helps. :slightly_smiling_face:</p>

**Comment by ElMameri7** [comment]
<p>Thanks for taking time to respond. I noticed that uppercase extension after i posted this and i fixed it but it still won’t work :confused:<br>
<img src="https://user-images.githubusercontent.com/70650915/181072491-06d219b1-b93f-49db-9875-6730490b0146.png" alt="error" data-base62-sha1="cjf7GcrGHTohDo7rSQWMTbIrL3E" width="536" height="454"></p>

**Comment by ElMameri7** [comment]
<p>oh nvm :sweat_smile: I fixed it i was so confused i only changed it in atom and not in the branch :sweat_smile: .<br>
Thanks aloooot :heart:</p>
